Base/usr/share/man/man5/SystemServer.md

```diff
@@ -24,8 +24,8 @@ describing how to launch and manage this service.
 * `Priority` - the scheduling priority to set for the service, either "low", "normal", or "high". The default is "normal".
 * `KeepAlive` - whether the service should be restarted if it exits or crashes. For lazy services, this means the service will get respawned once a new connection is attempted on their socket after they exit or crash.
 * `Lazy` - whether the service should only get spawned once a client attempts to connect to their socket.
-* `Socket` - a path to a socket to create on behalf of the service. For lazy services, SystemServer will actually watch the socket for new connection attempts. An open file descriptor to this socket will be passed as fd 3 to the service.
-* `SocketPermissions` - (octal) file system permissions for the socket file. The default permissions are 0600.
+* `Socket` - a comma-separated list of paths to sockets to create on behalf of the service. For lazy services, SystemServer will actually watch the socket for new connection attempts. See [socket takeover mechanism](#socket-takeover-mechanism) for details on how sockets are passed to services by SystemServer.
+* `SocketPermissions` - comma-separated list of (octal) file system permissions for the socket file. The default permissions are 0600. If the amount of socket permissions defined is less than the amount of sockets defined, then the last defined permission will be used for the remainder of the items in `Socket`.
```

Comment:
```
If the amount of socket permissions defined is less than the amount of sockets defined, then the last defined permission will be used for the remainder of the items in `Socket`.
```
Small nitpick: `amount` of socket permissions sounds strange, maybe 'level` or something else?

Reply:
What I tried to convey was that the following: ```ini
Socket=/tmp/sock1,/tmp/sock2,/tmp/sock3
SocketPermissions=600,660
```
would apply chmod `600` to `/tmp/sock1`, and chmod `660` to `/tmp/sock2` and `/tmp/sock2`. "Level" doesn't sound right to me, at least.

Reply:
```suggestion
* `SocketPermissions` - comma-separated list of (octal) file system permissions for the socket file. The default permissions are 0600. If the number of socket permissions defined is less than the number of sockets defined, then the last defined permission will be used for the remainder of the items in `Socket`.
```